### DISCUSSION TITLE
Use metaKey on macOS

### DISCUSSION THREAD

**Discussion by oeway** [DustinBrett/daedalOS#295] [Feature Requests]
Hi, it seems the current shortcuts for the file manager and the monaco editor are using ctrl+c, ctrl-v, ctrl-s as shortcuts, it would be great if we can use the meta (command) key instead on macOS.

**Comment by DustinBrett** [comment]
Thanks for this idea. I will look into considerations for Mac key bindings as I indeed only did the Windows ones (which is my main OS).
